internal/backend/sessions/sessioncalls/call.go

```diff
@@ -88,26 +88,19 @@ func (cs *calls) invoke(ctx context.Context, callv sdktypes.Value, args []sdktyp
 }
 
 // This is executed either in an activity (for regular calls) or directly in a workflow (for internal calls).
-func (cs *calls) executeCall(ctx context.Context, sessionID sdktypes.SessionID, seq uint32, executors *sdkexecutor.Executors) (debug any, attempt uint32, _ error) {
-	z := cs.z.With(zap.Uint32("seq", seq))
+func (cs *calls) executeCall(ctx context.Context, call sdktypes.SessionCallSpec, executors *sdkexecutor.Executors) (sdktypes.SessionCallAttemptResult, error) {
+	seq := call.Seq()
 
-	call, err := cs.svcs.DB.GetSessionCallSpec(ctx, sessionID, seq)
-	if err != nil {
-		return nil, 0, fmt.Errorf("db.get_call: %w", err)
-	}
+	sl := cs.sl.With(zap.Uint32("seq", seq))
```

Comment:
As agreed, `cs` should contains a logger, not a sugared logger.

```suggestion
	sl := cs.logger.With(zap.Uint32("seq", seq)).Sugar()
```

Reply:
done, but i don't think renaming to `logger` is right